onnxruntime/python/tools/quantization/qdq_loss_debug.py

```diff
@@ -344,7 +344,7 @@ def compute_signal_to_quantization_noice_ratio(
     right = numpy.concatenate(ylist).flatten()
 
     Ps = numpy.linalg.norm(left)
-    Pn = numpy.linalg.norm(left - right)
+    Pn = numpy.linalg.norm(left - right) + numpy.finfo("float").eps
```

Comment:
Curious to learn why this change is needed? It would be helpful to have a comment for the readers

Reply:
Pn is always positive per definition of vector norm. next line Pn is the denominator, this is to avoid divide by zero problem. This is actually pretty standard in numerical programming and it was stupid of me not to include this in the first version